### PR TITLE
Implementation finished

### PR DESCRIPTION
**Pull request by Ketch92**

### CODE REVIEW

**Review by mfedenko09 — APPROVED** [review]

File: logs/threadLogs.log [hunk 1]
```diff
@@ -0,0 +1,1002 @@
...
+ 21:32:23,455 INFO  MyThread:15 - Thread 993
+ 21:32:23,455 INFO  MyThread:15 - Thread 994
+ 21:32:23,455 INFO  MyThread:15 - Thread 995
+ 21:32:23,455 INFO  MyThread:15 - Thread 996
+ 21:32:23,455 INFO  MyThread:15 - Thread 997
+ 21:32:23,455 INFO  MyThread:15 - Thread 998
+ 21:32:23,455 INFO  MyThread:15 - Thread 999
+ 21:32:23,455 INFO  MyThread:15 - Thread 1000
+ 21:32:23,292 INFO  RunnableThead:15 - Runnable thread 511
+
```
Comment (RomanDubovskyi) [inline logs/threadLogs.log:1002]: add empty line at the end of the file

File: src/main/java/Counter.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+public class Counter {
+    private int counter;
+    
+    public Counter() {
+    }
```
Comment (RomanDubovskyi) [inline src/main/java/Counter.java]:
```suggestion
```
it'll be created by default

File: src/main/java/RunnableThead.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import org.apache.log4j.Logger;
+
+public class RunnableThead implements Runnable {
+    public static final int LIMIT = 1000;
```
Comment (RomanDubovskyi) [inline src/main/java/RunnableThead.java:4]:
```suggestion
    public static final int LIMIT = 100;
```
100 is enough

**Review by RomanDubovskyi — APPROVED** [review]

File: src/main/java/RunnableThead.java (hunk 3 above)
Comment (Ketch92) [reply src/main/java/RunnableThead.java:4]: It isn't enough for my PC. It quite offen runs the race for one thread. So to verify I increased the threshold

**Pull request merged**